syntax = "proto3";

package common;
option go_package = ".;commonpb";

message TrackMetadata {
  string title = 1;
  string artist = 2;
  string album = 3;
  string genre = 4;
  string userId = 5;
  string url = 6;
}

message StatusResponse {
  string status = 1;
}
